README.md

```diff
@@ -1,3 +1,5 @@
+**NOTE: This project is dead. See #559 for more information, including a V2.**
```

Comment:
I'd prefer "this repository is dead". The project as a whole encompasses V2 IMO.